server/zoom/www/cache/time_estimate_cache.py

```diff
@@ -263,10 +262,8 @@ def _check(self):
         :rtype: bool
         """
         try:
-            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.connect((self._host, 80))
-            s.close()
-            return True
+            r = requests.head(self.host, timeout=.1) # this might be too low.
```

Comment:
How will you know if this is too low? Worth making it configurable rather than hard-coded?